tests/foreman/cli/test_discoveryrule.py

```diff
@@ -614,20 +451,28 @@ def class_user_reader(self, class_user_password, class_org, class_location):
             logger.exception('Exception while deleting class scope user entity in teardown')
 
     @pytest.mark.tier2
-    def test_positive_create_rule_with_non_admin_user(
-        self, class_org, class_location, class_user_password, class_user_manager, class_hostgroup
+    def test_positive_create_view_and_delete_rule_with_non_admin_user(
+        self,
```

Comment:
I think we could've some update permissions assigned to non-admin user, then we could test update for it too, and complete its crud test for non-admin user, wdyt?

Reply:
Made it crud